app/static/js/slcNewProposal.js

```diff
@@ -78,7 +78,15 @@ $(document).ready(function(e) {
   });
 
   $("#cancelButton").on("click", function() {
-      window.location.replace($(this).val());
+    var cancelButtonContext = this
```

Comment:
let's just call this `cancelButton`. You could even convert it to jquery here, `$(this)`

Reply:
changed to cancelButton and moved the jquery to $(this)